ironfish/src/wallet/walletdb/accountValue.ts

```diff
@@ -38,6 +40,8 @@ export class AccountValueEncoding implements IDatabaseEncoding<AccountValue> {
     const id = reader.readVarString('utf8')
     const name = reader.readVarString('utf8')
     const spendingKey = reader.readBytes(KEY_LENGTH).toString('hex')
+    // view key is authorizing_key contactenated with nullifier_deriving_key
```

Comment:
Not super stoked about this, we could export another constant if you guys want, I can file a task for this if needed

Reply:
removed this